code/API_definitions/quality-on-demand.yaml

```diff
@@ -1024,3 +1110,20 @@ components:
           sessionId: '123e4567-e89b-12d3-a456-426614174000'
           qosStatus: 'UNAVAILABLE'
           statusInfo: 'DURATION_EXPIRED'
+
+    RETRIEVE_SESSIONS_EXAMPLE:
+      summary: List of QoS sessions for the device
+      description: A single QoS session for the device is available
+      value:
```

Comment:
Shouldn't that be an array of items?

Reply:
It is an array of items indeed, but with only one element